### PR TITLE
JBPM-6883 Stunner - can't move with bend points

### PR DESCRIPTION
**Pull request by tiagodolphine**
Here are the tests to https://github.com/kiegroup/lienzo-core/pull/57

@LuboTerifaj
@wmedvede (adding you to help on the review since @romartin is on PTO)

related:
kiegroup/kie-wb-common#2012
https://github.com/kiegroup/lienzo-core/pull/57

### CODE REVIEW

File: src/test/java/com/ait/lienzo/client/core/shape/wires/handlers/impl/WiresConnectorControlImplTest.java [hunk 1]
```diff
@@ -64,48 +77,98 @@
...
+        when(headMagnet.getMagnets()).thenReturn(magnets);
+        when(tailMagnet.getMagnets()).thenReturn(magnets);
+        when(headMagnet.getIndex()).thenReturn(1);
+        when(tailMagnet.getIndex()).thenReturn(1);
+        when(magnets.getWiresShape()).thenReturn(shape);
+        when(shape.getPath()).thenReturn(shapePath);
+
+        line = spy(new PolyLine(CP0, CP1, CP2, CP3, CP4));
+
+//        connector = spy(new WiresConnector(line,
```
Comment (LuboTerifaj) [inline src/test/java/com/ait/lienzo/client/core/shape/wires/handlers/impl/WiresConnectorControlImplTest.java]: I would remove code, which is commented out or add comment, why it is commented out.

**Comment by tiagodolphine** [comment]
@LuboTerifaj Done

**Comment by hasys** [comment]
Hi @manstis this PR is related to https://github.com/kiegroup/lienzo-core/pull/57 and ready to be merged. Thank you!

**Comment by manstis** [comment]
@hasys Please let me know when the full downstream on #2012 completes. Thanks.

**Pull request merged**